.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,49 @@
+name: Integration Tests
+
+on: [pull_request, push]
+
+jobs:
+  fprime-integration:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+        fprime-version: [NASA-v1.5.0, NASA-v1.5.1, NASA-v1.5.2, NASA-v1.5.3, v2.0.0, devel]
```

Comment:
I'm wondering if we need to integration test every patch release. Instead of testing 1.5.0-1.5.3 for example, I think we could probably just test 1.5.4. Whenever a new patch release comes up we can update these integration tests to test against the latest patch. What are your thoughts @LeStarch?

Reply:
I say we run against what is actively in use:

NASA-v1.5.3, v2.0.0, devel

Reply:
Sounds good!